contrib/buildahimage/stable/Dockerfile

```
@@ -23,6 +23,9 @@ RUN mkdir -p /var/lib/shared/overlay-images /var/lib/shared/overlay-layers /var/
 RUN echo build:2000:50000 > /etc/subuid \
  && echo build:2000:50000 > /etc/subgid
 
+VOLUME /var/lib/containers
+VOLUME /home/buildah/.local/share/containers
```

Should we use `build` here?
```suggestion
VOLUME /home/build/.local/share/containers
```

---

Yikes good catch @saschagrunert